src/test/java/com/olgaboiar/mint/ServerTest.java

```diff
@@ -46,4 +48,11 @@ void testServerAcceptsClientConnection () throws Exception {
         testClient.close();
     }
 
+    @Test
+    void testLoggerIsLoggingAfterServerStarts () throws Exception {
+        File file = new File("testLogs.txt");
+        String readLastLineOfTestLogsFile = new FileLoggerTest().readLastLine(file);
```

Comment:
Let's go over mocking tomorrow. You can merge in this PR without addressing this comment for now. 👍